doc/src/modules/solvers/solveset.rst

```diff
@@ -484,28 +488,28 @@ What will you do with the old solve?
  Hence, it's not yet a perfect replacement for old ``solve``. The ultimate
  goal is to:
 
- * Replace ``solve`` with ``solveset``, by the time solveset is
-   at least powerful as ``solve``, i.e. ``solveset`` does everything
-   that ``solve`` can do currently, and
+ * Replace ``solve`` with ``solveset`` once solveset is at least powerful as
+   ``solve``, i.e., ``solveset`` does everything that ``solve`` can do
+   currently, and
 
- * Eventually rename ``solveset`` to ``solve``.
+ * eventually rename ``solveset`` to ``solve``.
 
 
 How are symbolic parameters handled in solveset?
 ------------------------------------------------
 
- Solveset is in its initial phase of development as of now, so the
- symbolic parameters aren't handled well for all the cases, but some
- work has been done in this regard to depict our ideology towards
- symbolic parameters. As an example the solving of `|x| = n` for `x`
- where `n` is a symbolic parameter. Solveset returns the value of `x`
- considering the domain of the symbolic parameter `n` as well, i.e. :
+ Solveset is in its initial phase of development as of now, so the symbolic
+ parameters aren't handled well for all the cases, but some work has been done
+ in this regard to depict our ideology towards symbolic parameters. As an
+ example, consider the solving of `|x| = n` for real `x`, where `n` is a
+ symbolic parameter. Solveset returns the value of `x` considering the domain
+ of the symbolic parameter `n` as well:
 
- `([0, \infty) ∩ \{n\}) ∪ ((-\infty, 0] ∩ \{-n\})`.
+ .. math:: ([0, \infty) \cap \{n\}) \cup ((-\infty, 0] \cap \{-n\}).
 
- It simply means `n` is the solution only when it belongs to the ``Interval``
- `[0, \infty)` and `-n` is the solution only when `n` belongs to the ``Interval``
- `(- \infty, 0]`.
+ This simply means `n` is the solution only when it belongs to the
+ ``Interval`` `[0, \infty)` and `-n` is the solution only when `-n` belongs to
```

Comment:
I agree, it's wrong in master.

I think the reason that this is confusing is that it is mixing the solutions (`n`, `-n`) and domain restrictions on those solutions (`[0, oo)`, `(-oo, 0]`) in the same way. This is technically correct (it's a generalization of the idea [here](https://github.com/sympy/sympy/pull/10637/files#diff-eb270d82e76132827d2c4da08de6d801R329)), but using a ConditionSet would make it clearer that `{n, -n}` is the solution and `n >= 0` is the restriction on that solution. Another way to represent this sort of thing would be to use Piecewise.

Side note: this is very soon going to be hit by SymPy's inability to simplify basic inequality conditions.